README.md

```diff
@@ -33,12 +33,23 @@ The [amazon.aws](https://github.com/ansible-collections/amazon.aws) and [communi
 
 
 ### Installation
-Clone the collection repository.
 
-```shell
-  mkdir -p ~/.ansible/collections/ansible_collections/cloud/aws_troubleshooting
-  cd ~/.ansible/collections/ansible_collections/cloud/aws_troubleshooting
-  git clone https://github.com/redhat-cop/cloud.aws_troubleshooting .
```

Comment:
What if the user does not use the collection from AH?

Reply:
> What if the user does not use the collection from AH?

Hi @alinabuzachis , Based on the Slack conversations in #ansible-partners,  it seems like the consensus is to promote the use of Automation Hub. This seems to be the approach for other collections too.

Reply:
The Validated Content program does put a focus on platform experience, so I agree we should show AH instructions in the readme.